Source/Manager/IdentityManager.swift

```diff
@@ -259,7 +262,7 @@ public class IdentityManager: IdentityManagerProtocol {
             connection: identifier.connection,
             code: oneTimeCode,
             passwordlessToken: passwordlessToken,
-            scope: ["openid"]
+            scope: scopes + IdentityManager.defaultScopes
```

Comment:
Could you end up with getting duplicates here? Does it matter if you get duplicates?

Reply:
Hmm, good point. I fix.